server/src/main/java/org/opensearch/node/NodeService.java

```diff
@@ -159,7 +165,8 @@ public NodeInfo info(
             plugin ? (pluginService == null ? null : pluginService.info()) : null,
             ingest ? (ingestService == null ? null : ingestService.info()) : null,
             aggs ? (aggregationUsageService == null ? null : aggregationUsageService.info()) : null,
-            indices ? indicesService.getTotalIndexingBufferBytes() : null
+            indices ? indicesService.getTotalIndexingBufferBytes() : null,
+            searchPipelines ? (searchPipelineService == null ? null : searchPipelineService.info()) : null
```

Comment:
Super nit, please feel free to skip (just for readability):

```suggestion
            (searchPipelines && searchPipelineService != null) ? searchPipelineService.info()) : null
```

Reply:
I agree with your suggestion, but I'm also going for consistency with the convention used for all the other NodeInfo attributes.

If I fix this for searchPipelines, I would also like to update it for all the other fields too.

Honestly, this list of arguments is long enough that I feel like the best solution is a builder with all the "sub-infos" initialized as `null` and we do something like:
```
if (searchPipelines && searchPipelineService != null) {
  nodeInfoBuilder.searchPipelinesInfo(searchPipelineService.info());
}
```

Reply:
> I agree with your suggestion, but I'm also going for consistency with the convention used for all the other NodeInfo attributes.

Always try to leave the code in better shape than you found it :)